config/routes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  scope '/rails/action_mailbox', module: 'action_mailbox/ingresses' do
+    post '/ses/inbound_emails' => 'ses/inbound_emails#create',
+         as: :rails_ses_inbound_emails
```

Comment:
Is the `rails` prefix convention or necessary? Would prefer to remove.

Reply:
I'm pretty sure it's necessary to match action mailbox routes

Reply:
Like necessary in the sense of following action mailbox setup flow

Reply:
For reference, you can see the route structure: https://guides.rubyonrails.org/action_mailbox_basics.html

Reply:
I mean we want to follow the rails pattern*